lib/parsers/index.ts

```diff
@@ -34,7 +34,9 @@ export interface PkgTree {
   meta?: {
     nodeVersion: string;
   };
-  depType?: DepType;
+  labels?: {
+    depType: DepType;
+  };
   hasDevDependencies?: boolean;
   cyclic?: boolean;
   missingLockFileEntry?: boolean;
```

Comment:
are these already preserved btw? 
- `cyclic`
- `missingLockFileEntry`
- `cyclic`

Reply:
- Cycles are not yet supported in depGraph: https://github.com/snyk/dep-graph/blob/master/src/legacy/index.ts#L221-L224
- `missingLockFileEntry` - don't think this is preserved either